src/dotnet/commands/dotnet-pack/NuGet/Manifest.cs

```diff
@@ -8,7 +8,7 @@
 using System.Linq;
 using System.Xml.Linq;
 
-namespace NuGet
+namespace NuGet.Legacy
```

Comment:
I believe we will be, yes. @rohit21agrawal is undertaking the dotnet pack work (which is separate from my dotnet nuget work). This is why I've elected to use the more constistent namespace in the new code rather than adapt it to this code which will likely go away. In the meantime I need a way to resolve the conflict while waiting on the other work.